Dockerfile.build

```diff
@@ -0,0 +1,48 @@
+# Spreed WebRTC server Docker builder
+#
+# This Dockerfile creates a container which builds Spreed WebRTC as found in the
+# current folder, and creates a tarball which can be piped into another Docker
+# container for creating minimal sized Docker containers.
+#
+# First create the builder image:
+#
+#   ```
+#   docker build -t spreed-webrtc-builder -f Dockerfile.build .
+#   ```
+# Next run the builder container, piping its output into the creation of the
+# runner container. This creates a minimal size Docker image which can be used
+# to run Spreed WebRTC in production.
+#
+#   ```
+#   docker run --rm spreed-webrtc-builder | docker build -t spreed-webrtc -f Dockerfile.run -
+#   ```
+
+FROM ubuntu:xenial
+MAINTAINER Simon Eisenmann <simon@struktur.de>
+
+# Set locale.
+RUN locale-gen --no-purge en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Base build dependencies.
+RUN apt-get update && apt-get install -qy \
```

Comment:
To make future changes easier to track and get one-line diffs, I would rather write this as
```
RUN apt-get update && \
    apt-get install -qy \
        autoconf \
        automake \
        build-essential \
        git \
        golang \
        nodejs
```